fusor-ember-cli/app/templates/assign-nodes.hbs

```diff
@@ -25,9 +25,26 @@
             <div class="row">
                 <div class='col-md-7'>
                   {{#if nodes}}
-                    {{node-profile profile=profile nodes=nodes plan=model.plan doAssign=true assignRole="assignRole" unassignRole="unassignRole" editRole="editRole" setRoleCount="setRoleCount"  removeRole="removeRole"}}
+                    {{node-profile profile=profile
+                                   nodes=nodes
+                                   plan=model.plan
+                                   doAssign=true
+                                   assignRole="assignRole"
+                                   unassignRole="unassignRole"
+                                   editRole="editRole"
+                                   setRoleCount="setRoleCount"
+                                   removeRole="removeRole"
+                                   setRoleCountOnController="setRoleCountOnController"}}
```

Comment:
added this one line 
`setRoleCountOnController="setRoleCountOnController"`

It may not be intuitive as in simple examples, it would be `action="setRoleCountOnController"` but there are multiple actions passed to this component